pkg/docs/reference.md

```diff
@@ -0,0 +1,582 @@
+# rules_pkg - reference
```

Comment:
I wonder if this reference should be split up into multiple files as to perhaps be easier to navigate.  No need to do this now, though.

Reply:
I am not sure. I think it depends on how we want to present the common attributes. Do we pull them out to a first section, in which case I would keep them together, or copy them to each rule. For that distinct pages might be better.